src/api/providers/cerebras.ts

```diff
@@ -56,7 +56,7 @@ export class CerebrasHandler implements ApiHandler {
 
 		// Check if this is a reasoning model that uses thinking tags
 		const modelId = this.getModel().id
-		const isReasoningModel = modelId.includes("qwen") || modelId.includes("deepseek-r1-distill")
```

Comment:
Would this affect backward compatibility?

Reply:
No it would not the model actually needs to not be used because of lower context window